app/src/main/java/com/example/dashcarr/presentation/core/BaseFragment.kt

```diff
@@ -23,7 +26,8 @@ abstract class BaseFragment<VB: ViewBinding>(
         savedInstanceState: Bundle?
     ): View? {
         _binding = inflate.invoke(inflater, container, false)
-        showBottomNavigation(true)
+        Log.e("WatchingSomeStuff", "show = $showBottomNavBar")
```

Comment:
Is this necessary for "production code"?
Aleksander does review our code in P1, meaning we should remove all unnecessary code.

Reply:
done